tachyon/crypto/commitments/merkle_tree/field_merkle_tree/field_merkle_tree_unittest.cc

```diff
@@ -178,7 +178,7 @@ TEST_F(FieldMerkleTreeTest, CommitEitherOrder) {
   Tree tree = Tree::Build(hasher_, packed_hasher_, compressor_,
                           packed_compressor_, std::move(matrices));
 
-  std::vector<math::RowMajorMatrix<F>> matrices2 = {matrix, matrix2};
+  std::vector<math::RowMajorMatrix<F>> matrices2 = {matrix2, matrix};
```

Comment:
How did the test pass before?

Reply:
`CommitEitherOrder` is checking if the commit result is same even if the matrix order is different. Matrix order doesn't matter I think.

Reply:
The test checks if the results remain the same for inputs containing the same elements but in a different order. Since the input elements were ordered identically, the results should also be identical.